projects/admin/src/app/circulation/item/item.component.html

```diff
@@ -15,7 +15,7 @@
   along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
-<div *ngIf="item && !(item.loading || false) else loading"
+<div *ngIf="item && !(item.loading || false)"
```

Comment:
Simplify the test (just a suggestion)

Reply:
This need more time to change.  I propose to let this as it.